python/pyspark/sql/types.py

```diff
@@ -747,8 +747,8 @@ def jsonValue(self) -> Dict[str, Any]:
     def fromJson(
         cls,
         json: Dict[str, Any],
-        fieldPath: str,
-        collationsMap: Optional[Dict[str, str]],
+        fieldPath: str = "",
+        collationsMap: Optional[Dict[str, str]] = None,
```

Comment:
Can you add a test please?

Reply:
Extensive testing was added in #46280, this change is just so that the API for the `fromJson` method doesn't change which we missed in the previous PR